nas_ports/freenas/freenas-migrate113/Makefile

```diff
@@ -1,7 +1,8 @@
 # $FreeBSD$
 
 PORTNAME=	freenas-migrate113
-PORTVERSION=	2019112901
+PORTVERSION=	0.1
```

Comment:
Why the version change?

0.1 is smaller than 2019112901 so we need PORTEPOCH set if we really need that

Reply:
My copypasted script from zettarepl port did it :)